web-client/public/js/sliders.js

```diff
@@ -183,9 +183,7 @@ var sliderGroupController = function (sliderArray) {
 };
 
 // Gray Threshold Slider Settings
-var graySlider = document.getElementById("grayThresholdInput");
-
-var outputUpdate = function (val) {
-    // val = Math.round(val * 100) + '%';
-    document.querySelector("#grayThresholdValue").value = val;
+var outputUpdate = function () {
+    var value = Math.round(($("#grayThresholdInput").val() * 100));
+    $("#grayThresholdValue").html(value + "%");
```

Comment:
You can use `text` here instead of `html`, since the string you are setting doesn't have any HTML markup in it. As a general principle, you want to always use the "minimal" function that does the work that you need. In this case, since you don't need HTML to set the new value, you can opt for `text`:
```javascript
    $("#grayThresholdValue").text(value + "%");
```

Reply:
fixed, thanks for the feedback